namespaces/live.cloud-platform.service.justice.gov.uk/laa-apply-for-legalaid-staging/resources/pingdom.tf

```diff
@@ -2,16 +2,16 @@ provider "pingdom" {
 }
 
 resource "pingdom_check" "laa-apply-for-legalaid-staging" {
-  type                     = "http"
-  name                     = "Apply for legal aid staging - ping"
-  host                     = "staging.apply-for-legal-aid.service.justice.gov.uk"
-  resolution               = 1
-  notifywhenbackup         = true
-  notifyagainevery         = 0
-  url                      = "/ping"
-  encryption               = true
-  port                     = 443
-  tags                     = "businessunit_${var.business_unit},application_${var.application},component_ping,isproduction_${var.is_production},environment_${var.environment},infrastructuresupport_${var.application},laa,apply-for-legal-aid"
-  probefilters             = "region:EU"
-  integrationids           = [96995]
+  type             = "http"
```

Comment:
unrelated namespace changes -- could all files outside `namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-interventions-dev` be reverted, please?

Reply:
Sure it was not done by us but was by automatted code formatters